<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="xs12">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>30</y>
          <width>201</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>电缆故障检测仪</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>140</y>
          <width>361</width>
          <height>151</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>故障类型</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>故障距离</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="LX"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="JL"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>531</width>
          <height>191</height>
         </rect>
        </property>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>60</y>
           <width>101</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>波形图 </string>
         </property>
        </widget>
        <widget class="QPushButton" name="start">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>60</y>
           <width>101</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>开始检测</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>320</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>返回 </string>
        </property>
       </widget>
       <widget class="QChartView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>40</y>
          <width>281</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QChartView" name="graphicsView_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>281</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
